### DISCUSSION TITLE
Registering extra component on Vue app

### DISCUSSION THREAD

**Discussion by massimoliani** [directus/directus#17312] [Archive]
### Describe the Bug

Is more an improvement than a bug.

I'm developing custom interfaces I'm using on other Vue app too.
I'm developing using DevExtreme framework.

I would like to know if will be possible to access Vue app and register extra component one time only to avoid duplicated import os css and component for every interfaces.

Thank you.

### To Reproduce

...

### Errors Shown

...

### What version of Directus are you using?

9.8.0

### What version of Node.js are you using?

16.13.0

### What database are you using?

MySql 

### What browser are you using?

Chrome

### How are you deploying Directus?

locally or AWS

**Comment by paescuj** [comment]
Turned into a discussion since it's a question.
